extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/Lib485BridgeHandler.java

```diff
@@ -0,0 +1,150 @@
+                        return;
+                    }
+                }
+
+                if (socket.isConnected()) {
+                    receiverNodes.put(receiverNode, socket);
+                } else {
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+                    receiverNodes.put(receiverNode, null);
+                    return;
+                }
+            }
+            updateStatus(ThingStatus.ONLINE);
+        }
+    }
+
+    @Override
+    protected void closeConnection() {
+        for (IpNode receiverNode : receiverNodes.keySet()) {
+            Socket socket = receiverNodes.get(receiverNode);
+            if ((socket != null) && (!socket.isClosed())) {
+                try {
+                    socket.close();
+                } catch (IOException e) {
+                    logger.warn("Could not close socket {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                }
+            }
+            receiverNodes.put(receiverNode, null);
+        }
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+    }
+
+    @Override
+    protected void sendDmxData() {
+        if (this.thing.getStatus().equals(ThingStatus.ONLINE)) {
+            long now = System.currentTimeMillis();
+            universe.calculateBuffer(now);
+            for (IpNode receiverNode : receiverNodes.keySet()) {
+                Socket socket = receiverNodes.get(receiverNode);
+                if (socket.isConnected()) {
+                    try {
+                        socket.getOutputStream().write(universe.getBuffer());
+                    } catch (IOException e) {
+                        logger.warn("Could not send to {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                        closeConnection();
+                        return;
+                    }
+                } else {
+                    closeConnection();
+                    return;
+                }
+            }
+        } else {
+            openConnection();
+        }
+    }
+
+    @Override
+    protected void updateConfiguration() {
+
+        Configuration configuration = getConfig();
+
+        universe = new Universe(MIN_UNIVERSE_ID);
+
+        receiverNodes.clear();
+        if (configuration.get(CONFIG_ADDRESS) == null) {
+            receiverNodes.put(new IpNode("localhost:9020"), null);
+            logger.debug("sending to {} for {}", receiverNodes, this.thing.getUID());
+        } else {
+            for (IpNode receiverNode : IpNode.fromString((String) configuration.get(CONFIG_ADDRESS), DEFAULT_PORT)) {
+                receiverNodes.put(receiverNode, null);
+                logger.debug("sending to {} for {}", receiverNode, this.thing.getUID());
+            }
+        }
+        super.updateConfiguration();
+
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE);
```

Comment:
done, also I think that it's safe to assume the bridge is offline if it didn't connect before.